mylist.rb

```diff
@@ -0,0 +1,23 @@
+require('./myenumerable')
+
```

Comment:
- [ ] In ruby convention, it is recommended to use `require_relative` to import files within the same directory. `require` will be used to import  external files. Kindly work on it.